### PR TITLE
Feature/logging for filters

### PR DESCRIPTION
**Pull request by roxana-muresan**

### CODE REVIEW

File: core/src/test/java/de/valtech/aecu/core/groovy/console/bindings/impl/SimpleContentUpdateTest.java [hunk 1]
```diff
@@ -1,14 +1,14 @@
 package de.valtech.aecu.core.groovy.console.bindings.impl;
 
+import de.valtech.aecu.api.groovy.console.bindings.AecuBinding;
```
Comment (gruberrolandvaltech) [inline core/src/test/java/de/valtech/aecu/core/groovy/console/bindings/impl/SimpleContentUpdateTest.java]: No changes in this file, can be reset to devlop version

File: core/src/main/java/de/valtech/aecu/core/groovy/console/bindings/traversers/TraversData.java [hunk 2]
```diff
@@ -18,18 +18,18 @@
  */
 package de.valtech.aecu.core.groovy.console.bindings.traversers;
 
-import java.util.List;
-
-import javax.annotation.Nonnull;
+import de.valtech.aecu.api.groovy.console.bindings.filters.FilterBy;
+import de.valtech.aecu.core.groovy.console.bindings.actions.Action;
```
Comment (gruberrolandvaltech) [inline core/src/main/java/de/valtech/aecu/core/groovy/console/bindings/traversers/TraversData.java]: Same here, no changes

File: api/src/main/java/de/valtech/aecu/api/groovy/console/bindings/filters/FilterBy.java [hunk 3]
```diff
@@ -35,6 +35,6 @@
      * @param resource resource
      * @return matches
      */
-    boolean filter(@Nonnull Resource resource);
+    boolean filter(@Nonnull Resource resource, @Nonnull StringBuffer stringBuffer);
```
Comment (gruberrolandvaltech) [inline api/src/main/java/de/valtech/aecu/api/groovy/console/bindings/filters/FilterBy.java]: Breaking change, requires to change version to 2.0 in package-info.java.

Comment (roxana-muresan) [reply api/src/main/java/de/valtech/aecu/api/groovy/console/bindings/filters/FilterBy.java]: done

**Pull request merged**